### ISSUE TITLE
Cannot find "pthread.h"

### ISSUE THREAD

**Issue by wby238** [Microsoft/vcpkg#3482]
OS: x64 Windows7 SP1
I used `cmake "-DCMAKE_TOOLCHAIN_FILE=C:/src/vcpkg/scripts/buildsystems/vcpkg.cmake" -DCMAKE_BUILD_TYPE=Release ..`, and the output log is
```
-- Looking for pthread.h
-- Looking for pthread.h - not found
-- Found Threads: TRUE
```
The cmakelists.txt is:
```cmake
cmake_minimum_required(VERSION 3.5)
project(multithreads_project)

set(CMAKE_CXX_STANDARD 14)

find_package(Threads REQUIRED)

add_executable(multithreads_project main.cpp)
target_link_libraries(multithreads_project ${CMAKE_THREAD_LIBS_INIT})
```
I used `cmake --build .`, and the output log is:
```
“D:\code\multithreads-project\release\ALL_BUILD.vcxproj”(默认目标) (1) -> “D:\code\multithreads-project\release\multithreads_project.vcxproj”(默认目标) (3) ->(ClCompile 目标) ->  d:\code\multithreads-project\main.cpp(4): fatal error C1083: 无法打开包括文件: “pthread.h”: No such file or directory [D:\code\multithreads-project\release\multithreads_project.vcxproj]
```
How to fix it?

**Comment by jacobblock** [comment]
I run into a similar problem except threads is not reported as being found. Fails during configuration.

    vcpkg install boost:x86-windows pthreads:x86-windows
    ...
    cmake .. -DCMAKE_TOOLCHAIN_FILE="../vcpkg/scripts/buildsystems/vcpkg.cmake"

Boost is found, but pthread.h is not despite it being in the `installed/x86-windows/include`.

    -- Looking for pthread.h
    -- Looking for pthread.h - not found
    -- Could NOT find Threads (missing: Threads_FOUND)
    -- Could NOT find Threads (missing: Threads_FOUND)
    cmake : CMake Error at C:/Program Files/CMake/share/cmake-3.11/Modules/FindPackageHandleStandardArgs.cmake:137
    (message):
    At line:1 char:1
    + cmake .. -DCMAKE_TOOLCHAIN_FILE="../vcpkg/scripts/buildsystems/vcpkg. ...
    + ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
        + CategoryInfo          : NotSpecified: (CMake Error at ...:137 (message)::String) [], RemoteException
        + FullyQualifiedErrorId : NativeCommandError

       Could NOT find Threads (missing: Threads_FOUND)
    Call Stack (most recent call first):
      C:/Program Files/CMake/share/cmake-3.11/Modules/FindPackageHandleStandardArgs.cmake:378 
    (_FPHSA_FAILURE_MESSAGE)
       C:/Program Files/CMake/share/cmake-3.11/Modules/FindThreads.cmake:205 
     (FIND_PACKAGE_HANDLE_STANDARD_ARGS)
      vcpkg/scripts/buildsystems/vcpkg.cmake:247 (_find_package)
      cmake/UseThreads.cmake:3 (find_package)
      CMakeLists.txt:40 (include)

Seems like there is an issue in the toolchain file.

**Comment by JackBoosY** [comment]
Hi @wby238 , thanks for reporting this issue!
This issue has been fixed in #4081. Plz update vcpkg and rebuild pthread.
Thanks.
